dir.props

```diff
@@ -26,6 +26,11 @@
       <Value>True</Value>
     </AssemblyMetadata>
   </ItemGroup>
+  
+  <!-- Temporary adding this property so that Linux builds will target the net45 version of the task library instead of the .Net Core version -->
```

Comment:
The comment is wrong, it should read 'non-windows', I'll fix that.

Sure, in the latest buildtools nuget package (00123) we have two different versions of the Build Tasks library, one that runs in .NetCore and one that runs in .Net 4.5. Environments that use .Net 4.5 to build, require loading the tasks from the latter. Linux is an example for this, since it uses mono msbuild to build, which only supports .Net 4.5 so if we were to try to load the .Net Core version of the library it would fail. This will be shortly fixed in Linux since we will move to use the .Net Core version of msbuild instead of mono, which is why this variable is only temporary.